Samples/VirtualCamera/VirutalCamera_TestSuite.sln

```diff
@@ -0,0 +1,124 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00

```

Comment:
This file has a typo in its name (Vi**rut**al == Vi**rtu**al

Reply:
fix, other places in the project too.